<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QToolButton" name="btBack">
     <property name="text">
      <string>←</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/left.png</normaloff>icons/left.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="11">
    <widget class="QToolButton" name="btDelete">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QToolButton" name="btInsert">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="btHome">
     <property name="text">
      <string>⌂</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/home.png</normaloff>icons/home.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="12">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="btReload">
     <property name="text">
      <string>↻</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/reset.png</normaloff>icons/reset.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="5">
    <widget class="QLineEdit" name="txUrl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="btForward">
     <property name="text">
      <string>→</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/right.png</normaloff>icons/right.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QToolButton" name="btGoUrl">
     <property name="text">
      <string>Go</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/go.png</normaloff>icons/go.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
